scripts/nn/layers/elu.dml

```diff
@@ -0,0 +1,60 @@
+#-------------------------------------------------------------
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+#-------------------------------------------------------------
+
+/*
+ * Exponential Linear Units (ELUs) nonlinearity layer.
+ */
+
+forward = function(matrix[double] X, int alpha)
+    return (matrix[double] out) {
+  /*
+   * Computes the forward pass for a ELUs nonlinearity layer.
+   *
+   * Performs an element-wise evaluation of `f(input) = max(0, input) + min(0, alpha ∗ (exp(input) − 1))`.
```

Comment:
Can you check that the line length here is <= 72?  For the `nn` library, I've kept all docstring lines <= 72 characters in length.  Also, can you (1) reference the ELU paper (https://arxiv.org/abs/1511.07289v1), and (2) add documentation that this formulation matches the if/else formula from the paper `(f(x) = x if x ≥ 0 else α (exp(x) − 1))`.